db/migrate/20190528185128_drop_devices.rb

```diff
@@ -0,0 +1,9 @@
+class DropDevices < ActiveRecord::Migration[5.2]
```

Comment:
no need of this migrations, since u adding table in this PR
meaning - what's the point of adding and at the same time removing of the table?
just delete migration that adds the table